enes-homework-week3-moreJS.js

```diff
@@ -0,0 +1,200 @@
+//Q:More JS
+
+//Q1
+
+function sum(a,b,c) {
+  return (a+b+c);
+};
+
+console.log(sum(1,2,3));
+
+//Q2
+
+function colorCar(str) {
+  console.log("a " + str + " car" );
+}
+
+const str1 = "red";
+
+console.log(colorCar(str1));
+
+
+//Q3
+const obj1 = {
+  name: "Jason",
+  surname: "Born",
+  age: 33,
+};
+
+function printOutPropertiesAndValues (obj) {
+  for (let key in obj) {
+    console.log(key);
```

Comment:
Hi Enes, this function works fine. However, if you remember from our class, I mentioned that you can try to use `Object.keys` instead of a `for-in` loop. You can perhaps try to google the difference between the two approaches and see if it makes sense. Otherwise, I think this is going to be covered in JS3. 🙂

Reply:
Thanks again. I made some readings and learnt to some extent the difference.